config/src/config/test_data/random.complete.node.config.toml

```diff
@@ -67,26 +67,26 @@ dir = "libradb/db"
 grpc_max_receive_len = 100000000
 
 [test.account_keypair]
-private_key = "a9ccbb14534fa25b8e2c8062f82592ca4a6ecd228895ef48a21faf480f4fdb9f"
-public_key = "8a09af25da4e547c7192a0f7f05c2b8db94af316a5b599bd2cff8797c28ae68b"
+private_key = "f6b898412f4ab061943167c1e23efaa2ba98e345a093f0b06da13bffdbd4b2c7"
+public_key = "5cea31ed5918cbdac5f5c4e782d295c0b3033ed5d0eac941e8ea8bb0ab4480a3"
 [test.consensus_keypair]
-private_key = "87572b6c1ac0bc9f5245d04f7963c35db0713f0f08685f038d55f8c22a5c45b9"
-public_key = "7fb83ee67e2b16f89510cdd7c061b32a482d68465d24df70acc65b074b3f7862"
+private_key = "f2b916c8b1bf6123854d2fca5384f128199f7a1b2f847316481429b4b35238cc"
+public_key = "ca68c65c24a94c3aca0626e573dbdfe7e799a2565e9051b634fec1a42d3baaf9"
 
 [validator_network]
-peer_id = "afd41847853f81de4b37cd030195b25a"
+peer_id = "31893204fa402143c11b26ce8a89ea1d"
 listen_address = "/ip4/0.0.0.0/tcp/6180"
 advertised_address = "/ip4/127.0.0.1/tcp/6180"
 discovery_interval_ms = 1000
 connectivity_check_interval_ms = 5000
 enable_noise = true
 enable_remote_authentication = true
 network_peers_file = ""
-seed_peers_file = "afd41847853f81de4b37cd030195b25a.seed_peers.toml"
+seed_peers_file = "31893204fa402143c11b26ce8a89ea1d.seed_peers.toml"
 
 [validator_network.network_keypairs.signing_keys]
-private_key = "55f8f8956dde49b412e105460193381bf3fca57565a8a096e60ac7bdd6a0953f"
-public_key = "f5c974b9ead1ca6a6d5d93eb3bd5906e2b7cf5f710c8a7c86c8c6e918f42c4b1"
+private_key = "66dd107034b4582a2ef42c5e1ea475f2fea477a10a9f1d75b3635243b2506b32"
```

Comment:
What’s the reason for changing these keys?

Reply:
We changed the type of rng that generates them (`EntropyRng` had to [go away](https://github.com/rust-random/rand/commit/8cfe2101c57aae204706a7d03e9d90233eb962b2)).